lib/Models/Workflows/SelectableDimensionWorkflow.ts

```diff
@@ -25,6 +31,29 @@ export default interface SelectableDimensionWorkflow {
   selectableDimensions: SelectableDimensionWorkflowGroup[];
 }
 
+namespace SelectableDimensionWorkflow {
+  /**
+   * Runs a selectable dimension workflow which is a workflow for a workbench item.
+   *
+   * @param viewStateOrTerria - The {@link ViewState} or {@link Terria} instance
+   * @param workflow - A {@link SelectableDimensionWorkflow} instance
+   */
+  export function runWorkflow(
+    viewStateOrTerria: ViewState | Terria,
```

Comment:
Why `ViewState | Terria`. It can be confusing for an end developer to decide when to use `viewState` and when to use `terria`. So I think, as much as possible, our APIs must accept `viewState`. But models do not have access to `viewState` so for this method to be usable from models we have to also accept `terria`.